tst/EnergyPlus/unit/CurveManager.unit.cc

```diff
@@ -742,3 +744,1014 @@ TEST_F(EnergyPlusFixture, CSV_CarriageReturns_Handling)
+                                                "  1,                                      !- Coefficient6 z",
+                                                "  0,                                      !- Minimum Value of v {BasedOnField A2}",
+                                                "  1,                                      !- Maximum Value of v {BasedOnField A2}",
+                                                "  0,                                      !- Minimum Value of w {BasedOnField A2}",
+                                                "  1,                                      !- Maximum Value of w {BasedOnField A2}",
+                                                "  0,                                      !- Minimum Value of x {BasedOnField A3}",
+                                                "  1,                                      !- Maximum Value of x {BasedOnField A3}",
+                                                "  0,                                      !- Minimum Value of y {BasedOnField A4}",
+                                                "  1,                                      !- Maximum Value of y {BasedOnField A4}",
+                                                "  0.8,                                    !- Minimum Value of z {BasedOnField A5}",
+                                                "  0.5,                                    !- Maximum Value of z {BasedOnField A5}",
+                                                "  ,                                       !- Minimum Curve Output {BasedOnField A4}",
+                                                "  ,                                       !- Maximum Curve Output {BasedOnField A4}",
+                                                "  Dimensionless,                          !- Input Unit Type for v",
+                                                "  Dimensionless,                          !- Input Unit Type for w",
+                                                "  Dimensionless,                          !- Input Unit Type for x",
+                                                "  Dimensionless,                          !- Input Unit Type for y",
+                                                "  Dimensionless;                          !- Input Unit Type for z",
+                                            }),
+                                            "Minimum Value of z [0.80] > Maximum Value of z [0.50]"}),
+                         [](const testing::TestParamInfo<CurveManagerValidationFixture::ParamType> &info) -> std::string {
+                             auto object_name = info.param.object_name;
+                             std::replace_if(object_name.begin(), object_name.end(), [](char c) { return !std::isalnum(c); }, '_');
+                             return object_name + "_" + info.param.tested_dim;
+                         });
```

Comment:
instantiate the parametrized test. Test was prepared via https://gist.github.com/jmarrec/f86c5dd911568cb08aa293479d1a59a3